db/projects/litecoin/project.yaml

```diff
@@ -1,15 +1,46 @@
-name: litecoin
+name: Litecoin
+start_date: '2011–10-07’
```

Comment:
Failuje to kvuli tomuhle, musi se pouzivat dvojite uvozovky.
Takze nejlepsi bude pouzivat dvojite uvozovky vsude a je to
```yaml
start_date: "2011–10-07"
```

Reply:
Zkus ty cisla prepsat znovu rucne, je to v nejakych divnych znacich